facebook/__init__.py

```diff
@@ -335,13 +334,14 @@ def __init__(self, result):
             # OAuth 2.0 Draft 00
             try:
                 self.message = result["error"]["message"]
+                if not self.type:
+                    self.type = result["error"].get('type', '')
```

Comment:
Earlier, `self.type` is defined as `result["error_code"]`. Is it possible that this change could overwrite that?